### ISSUE TITLE
"Save and Close" doesn't work any more

### ISSUE THREAD

**Issue by muzimuzhi** [texstudio-org/texstudio#3063]
### Environment
- TeXstudio: 4.5.2beta4
- Qt: 6.5.0
- OS: macOS 12.6.3
- TeX distribution: irrelevant

### Expected behavior

The "Save and Close" and "Close without Saving" do what's expected.
![image](https://user-images.githubusercontent.com/6376638/230544700-9db364cf-1556-453d-ab4a-ca94c7d6e7eb.png)

### Actual behavior
Nothing happens

### How to reproduce

- Create an empty tab, type in a (dummy) tex document, and compile
- Close this tab, in the pop-up window, select either "Save and Close" or "Close without Saving"

A dummy tex document for testing:
```tex
\documentclass{article}
\begin{document}
content
\end{document}
```

Menu "File - Save" or "File - Save as..." still works.

**Comment by sunderme** [comment]
works on linux.

**Comment by mbertucci47** [comment]
Also works on Windows

**Comment by sunderme** [comment]
qt6.5/OSX bug

**Comment by muzimuzhi** [comment]
I confirm this is fixed in 4.5.2rc1. Many thanks!
